pkg/pool-stable-phantom/test/StablePoolAmplification.test.ts

```diff
@@ -0,0 +1,300 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { BigNumber, Contract } from 'ethers';
+
+import { sharedBeforeEach } from '@balancer-labs/v2-common/sharedBeforeEach';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { bn } from '@balancer-labs/v2-helpers/src/numbers';
+import {
+  advanceTime,
+  advanceToTimestamp,
+  currentTimestamp,
+  DAY,
+  setNextBlockTimestamp,
+} from '@balancer-labs/v2-helpers/src/time';
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+import { deploy } from '@balancer-labs/v2-helpers/src/contract';
+import Vault from '@balancer-labs/v2-helpers/src/models/vault/Vault';
+
+describe('StablePoolAmplification', () => {
+  let owner: SignerWithAddress, admin: SignerWithAddress, other: SignerWithAddress;
+  let pool: Contract;
+
+  const AMP_PRECISION = 1e3;
+  const AMPLIFICATION_PARAMETER = bn(200);
```

Comment:
```suggestion
  const INITIAL_AMPLIFICATION_PARAMETER = bn(200);
```

Reply:
We should also have the tests for initial values (checked in the constructor) here

Reply:
[a67ebca](https://github.com/balancer-labs/balancer-v2-monorepo/pull/1559/commits/a67ebcae8e1f824a824edf3642c879f24bb9bed8)